src/sagemaker_sklearn_extension/feature_extraction/sequences.py

```diff
@@ -70,6 +75,14 @@ class TSFeatureExtractor(BaseEstimator, TransformerMixin):
         'efficient': extract 781 tsfresh features, namely all of them except for the expensive to compute ones
         'all': extract all 787 tsfresh features
 
+    extraction_seed : int (default = 0)
+        Random seed used to choose subset of features, when expansion control does not allow to include all features
+
+    sequences_lengths_q25 : list of ints (default = None)
```

Comment:
Why isn't `sequences_lengths_q25` calculated in the fit stage?

Reply:
We need global stats here, in case if we want to train on subsample, but make decision based on sequences_lengths_q25 computed for the entire dataset.